frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx

```diff
@@ -5,44 +5,46 @@ import upArrowIcon from 'assets/uparrow.png';
 import messages from './displayMessages';
 import './styles.scss';
 
-interface ImageUploadFieldProps {
+interface ModalImageInputProps {
   customUploadMessage: any;
-  updateImage: Function;
-  clearError: Function;
-  parentMessages?: any;
-  recommendationTextId?: string;
+  clearError: () => void;
   error?: string;
+  innerPreview?: string;
   loading?: boolean;
+  parentMessages?: any;
+  recommendationTextId?: string;
   reset?: Function;
+  updateImage: Function;
+}
+
+interface ImageUploadFieldProps extends ModalImageInputProps {
   tooltipTextId?: string;
   tooltipImage?: string;
-  innerPreview?: string;
-  children?: React.ReactNode;
 }
 
 interface Image {
   name?: string;
 }
 
-export const ImageUploadField: React.FC<ImageUploadFieldProps> = (
-  props: ImageUploadFieldProps
+export const ModalImageInput: React.FC<ModalImageInputProps> = (
```

Comment:
This component is an extraction of just the actual input parts of `ImageUploadField`.